fw/rbcx-coprocessor/src/ControlLink.cpp

```diff
@@ -17,9 +20,13 @@
 static DMA_HandleTypeDef dmaRxHandle;
 static DMA_HandleTypeDef dmaTxHandle;
 static rb::CoprocCodec codec;
-static rb::CoprocLinkParser<EspMessage, &EspMessage_msg, &codec> parser;
+static rb::CoprocLinkParser<CoprocReq, &CoprocReq_msg, &codec> parser;
 static ByteFifo<512> rxFifo;
-static std::array<uint8_t, codec.MaxFrameSize> txFrameBuf;
+
+// Encode TX frame in txEncodeBuf, push to txMessageBuf, move to txDmaBuf and send via DMA.
```

Comment:
Úplně mi tento komentář není jasný. Znamená to že:
Data přijdou do `txEncodeBuf`, pak se načtou do `txMessageBuf` a z něj jdou do `txDmaBuf` z kterého se přes DMA odesílají?

Reply:
jj